maven-plugin/src/main/java/org/openmrs/maven/plugins/Run.java

```diff
@@ -270,6 +271,8 @@ private void runInFork(Server server) throws MojoExecutionException, MojoFailure
 		}
 
 		System.out.println("\nForking a new process... (use -Dfork=false to prevent forking)\n");
+		System.setProperty("MAVEN_OPTS", mavenOpts);
+		System.out.println("\n" + "Using MAVEN_OPTS: " + mavenOpts + "\n");
```

Comment:
Please remove since lines 292,293 do it already. Also the output is done in RunTomcat.